### PR TITLE
Add data attributes to breadcrumbs

### LINKED ISSUE #37: Add data attribute to link
Hi,

On one of my links, I need to add a data attribute like below:

data-turbo-frame="_top"

How can I add this?

### PR DESCRIPTION
**Pull request by SpaYco**
Closes #37

### CODE REVIEW

File: lib/gretel/renderer.rb [hunk 1]
```diff
@@ -273,6 +273,10 @@ def join_classes(*classes)
         clazz.blank? ? nil : clazz
       end
 
+      def data_attributes
+        options[:data_attributes] || options[:data] || {}
```
Comment (kzkn) [inline lib/gretel/renderer.rb:277]: why `options[:data]` is needed?

**Review by kzkn — COMMENTED** [review]
@SpaYco 
Thanks for the pull request!

**Comment by kzkn** [comment]
@SpaYco 
I was thinking about naming parameters. Parameters that assign attributes to links are prefixed with `link_` prefix. I feel that parameters that give data attributes to links should follow the convention.
How about naming the parameter `link_data`?

**Comment by SpaYco** [comment]
@kzkn Sounds great! I updated the code :)

**Review by kzkn — APPROVED** [review]
Thanks!

**Pull request merged**